package.json

```diff
@@ -71,18 +71,18 @@
     "eslint-plugin-vue": "^8.7.1",
     "eslint-plugin-vuetify": "^1.1.0",
     "flush-promises": "^1.0.2",
-    "happy-dom": "^10.5.2",
+    "jsdom": "^22.1.0",
```

Comment:
Kind of a collateral fix (since another UI was done recently), but there are also fewer unit test failures to fix with this environment compare with happy-dom.